### PR TITLE
Admin can mark off a volunteer

### PR DESCRIPTION
**Pull request by 13204767**
Administrators can mark off volunteers.

### CODE REVIEW

**Review by jiak1 — APPROVED** [review]

File: frontend/src/components/attendanceList.tsx [hunk 1]
```diff
@@ -117,6 +117,7 @@ export default function AttendanceListModal({
      */
 
     const displayAttendanceList = (attendanceList: UserShiftAttendaceSummary[]) => {
+        console.log(attendanceList);
```
Comment (jiak1) [inline frontend/src/components/attendanceList.tsx:120]: I don't think we need this console.log

**Review by jiak1 — CHANGES_REQUESTED** [review]

File: backend/src/User/user.controller.ts [hunk 2]
```diff
@@ -353,8 +353,12 @@ export const setCompleteShift = async (req: Request, res: Response) => {
         // This should be adjusted to check if the user is a supervising volunteer not admin
         // Will likely have to make DB adjustments for this to identify if user is supervisor
         const isAdmin = userObj?.isAdmin || false;
-        const sessionUserId = userObj._id;
-        if (!isAdmin && sessionUserId.toString() !== req.params.userid) {
+        let targettedUserId = userObj._id;
+        console.log(userObj);
```
Comment (jiak1) [inline backend/src/User/user.controller.ts:357]: I don't think we need this console.log

**Review by jiak1 — CHANGES_REQUESTED** [review]

**Review by 13204767 — COMMENTED** [review]
Removed console logs

**Review by jiak1 — APPROVED** [review]

**Comment by jiak1** [comment]
@13204767 is this right to be merged?

**Comment by 13204767** [comment]
Yeah my bad

**Pull request merged**